src/patterns/addresses/example-address-input-non-editable-confirm-address.njk

```diff
@@ -21,7 +21,7 @@ layout: ~
             "ariaLabel": 'Back',
             "itemsList": [
                 {
-                    "url": '/patterns/addresses/examples/address-input-non-editable',
+                    "url": '/patterns/addresses/examples/example-address-input-non-editable',
```

Comment:
I have checked this within the local instance of the service manual and the link as it is doesn't work. 
if you only add this in the url parameter it will work. 

"example-address-input-non-editable-register-address"